src/components/layout/modal.js

```diff
@@ -66,16 +66,15 @@ const Modal = ({ children, isOpen, onClose, ariaLabel, initialFocus }) => {
   return (
     <div
       className={`su-modal
-      su-fixed su-w-full su-h-full su-top-0 su-left-0 su-items-center
-      su-justify-center su-z-50 su-pointer-events-none
+      su-fixed su-w-full su-h-full su-top-0 su-left-0 su-items-center su-justify-center su-z-50
       ${isOpen ? "su-flex" : "su-hidden"}
     `}
       aria-label={ariaLabel}
       aria-hidden={isOpen ? "false" : "true"}
       role="dialog"
       tabIndex="-1"
     >
-      <div className="su-absolute su-w-full su-h-full su-bg-saa-black su-opacity-95 su-pt-70">
+      <div className="su-absolute su-w-full su-h-full su-opacity-95 su-bg-saa-black su-pt-70 su-overflow-auto">
```

Comment:
```suggestion
      <div className="su-absolute su-w-full su-h-full su-bg-opacity-95 su-bg-saa-black su-pt-70 su-overflow-auto">
```
I prefer that we use `su-bg-opacity-95` if we only need to change the opacity of the background colors. `su-opacity-95` changes opacity on the content also. It's not noticeable because it's 95%, but let's try to follow good practice :)
https://tailwindcss.com/docs/background-opacity